server/util.go

```diff
@@ -222,7 +222,15 @@ func interceptConfigs(rootViper *viper.Viper, customAppTemplate string, customCo
 		conf.RPC.PprofListenAddress = "localhost:6060"
 		conf.P2P.RecvRate = 5120000
 		conf.P2P.SendRate = 5120000
-		conf.Consensus.TimeoutCommit = 5 * time.Second
+
+		defaultCometCfg := tmcfg.DefaultConfig()
```

Comment:
This change allows custom value of `TimeoutCommit` [here](https://github.com/dydxprotocol/v4/blob/e290bc0b2f47e89ac47b8b85395d2b02278aaae8/cmd/dydxprotocold/cmd/config.go#L64) to take affect.